<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replica</class>
 <widget class="QWidget" name="Replica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1392</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>79</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="portnumLabel">
       <property name="text">
        <string>Port Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="portnumBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="initButton">
       <property name="text">
        <string>Initialize Replica</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>53</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="box1">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="box2">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="box3">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="box4">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QDoubleSpinBox" name="box5">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDoubleSpinBox" name="box6">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QDoubleSpinBox" name="box7">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QDoubleSpinBox" name="box8">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QDoubleSpinBox" name="box9">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QDoubleSpinBox" name="box10">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QDoubleSpinBox" name="box11">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QDoubleSpinBox" name="box12">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="12">
      <widget class="QDoubleSpinBox" name="box13">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="13">
      <widget class="QDoubleSpinBox" name="box14">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="14">
      <widget class="QDoubleSpinBox" name="box15">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="15">
      <widget class="QDoubleSpinBox" name="box16">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> pi</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label4">
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label5">
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label6">
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label8">
       <property name="text">
        <string>8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLabel" name="label9">
       <property name="text">
        <string>9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLabel" name="label10">
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QLabel" name="label11">
       <property name="text">
        <string>11</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="11">
      <widget class="QLabel" name="label12">
       <property name="text">
        <string>12</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="12">
      <widget class="QLabel" name="label13">
       <property name="text">
        <string>13</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="13">
      <widget class="QLabel" name="label14">
       <property name="text">
        <string>14</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="14">
      <widget class="QLabel" name="label15">
       <property name="text">
        <string>15</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="15">
      <widget class="QLabel" name="label16">
       <property name="text">
        <string>16</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>79</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>portnumBox</tabstop>
  <tabstop>initButton</tabstop>
  <tabstop>box1</tabstop>
  <tabstop>box2</tabstop>
  <tabstop>box3</tabstop>
  <tabstop>box4</tabstop>
  <tabstop>box5</tabstop>
  <tabstop>box6</tabstop>
  <tabstop>box7</tabstop>
  <tabstop>box8</tabstop>
  <tabstop>box9</tabstop>
  <tabstop>box10</tabstop>
  <tabstop>box11</tabstop>
  <tabstop>box12</tabstop>
  <tabstop>box13</tabstop>
  <tabstop>box14</tabstop>
  <tabstop>box15</tabstop>
  <tabstop>box16</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
